cost-analyzer/templates/cost-analyzer-oidc-config-map-template.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.oidc }}
+{{- if .Values.oidc.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cost-analyzer.fullname" . }}-oidc
+  labels:
+    {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
+data:
+{{- $root := . }}
+    oidc.json: '{{ toJson .Values.oidc }}'
```

Comment:
Won't this put the `clientSecret` into the JSON?

Reply:
Yep... fixing to only contain the relevant fields.